test/cpp/api/functional.cpp

```diff
@@ -997,7 +997,7 @@ TEST_F(FunctionalTest, GELU) {
   const auto x = torch::linspace(-3.0, 3.0, 100);
   const auto y_exp = x * 0.5 * (1.0 + torch::erf(x / std::sqrt(2.0)));
   const auto y = F::gelu(x);
-  ASSERT_TRUE(torch::allclose(y, y_exp));
+  ASSERT_TRUE(torch::allclose(y, y_exp, 1e-3, 0));
```

Comment:
Tell me more about the precision difference?

Reply:
Here is a mistake. Should be "ASSERT_TRUE(torch::allclose(y, y_exp, 1.3e-06, 1e-05));".